<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskInputDialog</class>
 <widget class="QDialog" name="TaskInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>789</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>994</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <property name="windowIcon">
   <iconset resource="backgroundsrc.qrc">
    <normaloff>:/images/graylogo.png</normaloff>:/images/graylogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#TaskInputDialog{
background: #c3c3c3;
border-image:url(:/images/finalBackground.jpg)
}

QPushButton, QMenuButton, QStatusBar, QTextEdit, QLabel, QDateTimeEdit, QSpinBox{
font-family: &quot;Roboto Black&quot;;
font-size: 42px;
}

QPushButton{
background-color:white;
padding: 10px;
}

QPushButton:hover{
background-color: #eb3f79;
color: white;
}

QRadioButton::indicator::unchecked{
border: 3px solid;
border-color: black;
border-radius: 6px;
background-color: white;
}

QRadioButton::indicator::checked{
border: 3px solid;
border-color: black;
border-radius: 6px;
background-color: #eb3f79
}
</string>
  </property>
  <widget class="QWidget" name="Content" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>921</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>861</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto Black</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Task Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_dueDate">
       <property name="minimumSize">
        <size>
         <width>182</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Roboto Black</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Due Date</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="input_dueDate">
       <property name="font">
        <font>
         <family>Roboto Black</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>11</month>
         <day>22</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>11</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>M/d/yyyy h:mm AP</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_hrRequired">
       <property name="font">
        <font>
         <family>Roboto Black</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time Needed               (1-1440 Mins)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="input_hrR_hr">
       <property name="font">
        <font>
         <family>Roboto Black</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1440</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="input_name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>6</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="topPanel" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1029</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1029</width>
     <height>131</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#topPanel{
border-image:url(:/images/mainHeader.png)
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Title{
font-weight: bold;
font-size: 52px;
}</string>
   </property>
   <property name="text">
    <string>Input Tasks</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>700</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="backgroundsrc.qrc"/>
 </resources>
 <connections/>
</ui>
